x-pack/plugins/ml/public/alerting/jobs_health_rule/register_jobs_health_alerting_rule.ts

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { lazy } from 'react';
+import { i18n } from '@kbn/i18n';
+import { TriggersAndActionsUIPublicPluginSetup } from '../../../../triggers_actions_ui/public';
+import { PluginSetupContract as AlertingSetup } from '../../../../alerting/public';
+import { ML_ALERT_TYPES } from '../../../common/constants/alerts';
+import { MlAnomalyDetectionJobsHealthRuleParams } from '../../../common/types/alerts';
+
+export function registerJobsHealthAlertingRule(
+  triggersActionsUi: TriggersAndActionsUIPublicPluginSetup,
+  alerting?: AlertingSetup
+) {
+  triggersActionsUi.alertTypeRegistry.register({
+    id: ML_ALERT_TYPES.AD_JOBS_HEALTH,
+    description: i18n.translate('xpack.ml.alertTypes.jobsHealthAlertingRule.description', {
+      defaultMessage: 'Alert when anomaly detection jobs experiencing realtime issues.',
```

Comment:
Not all of the conditions will affect real-time jobs, so I think this should read `Alert when anomaly detection jobs experience operational issues.`

Reply:
Fixed in https://github.com/elastic/kibana/pull/106084/commits/9c0b3ecfa61edfea97ca0881482ca69af5ebd954